### ISSUE TITLE
Panic on font name length

### ISSUE THREAD

**Issue by JoBrad** [jwilm/alacritty#2234]
Which operating system does the issue occur on?
Windows 10 (Powershell Core)

I installed Alacritty using Scoop, per the instructions from Alacritty's Readme file.

Problem: If the family value for a font is longer than 31 characters, Alacritty fails on startup with a panic. 

Example config below that causes this issue:
```
font:
  normal:
    family: Roboto Mono Medium for Powerline
```

When Alacritty is started, the following error occurs:
```
---------------------------
Alacritty: Runtime Error
---------------------------
panicked at 'Font length must me smaller than 31', C:\Users\travis\.cargo\registry\src\github.com-1ecc6299db9ec823\font-loader-0.6.0\src\win32.rs:86:17

Press Ctrl-C to Copy
---------------------------
OK   
---------------------------
```

That path doesn't exist on my computer - not sure if that matters, or not. 

Changing the family name to a value shorter than 31 characters does not cause the issue. This value does not cause a panic, for example: 
```
font:
  normal:
    family: Roboto Mono Light for Powerline
```

**Comment by chrisduerr** [comment]
It would be interesting to see if this can be reproduced with https://github.com/jwilm/alacritty/pull/2224. Since that uses the native Windows font stack it should hopefully resolve the problem.

**Comment by billti** [comment]
Hitting this too, which is a pain as most of the Powerline fonts have really long names (fortunately the "Anonymice Powerline" is decent enough and less than 32 chars.

![font-string](https://user-images.githubusercontent.com/993909/57474824-e57e1280-7247-11e9-97f9-cd69a0d894d3.png)

I also notice if you select a non-TrueType font then launching just dies with no message. If you know where to look for the log (%LOCALAPPDATA%\Temp) you will find the below, but some type of dialog like the above when failing to load would be helpful.

```
[2019-05-09 10:35] [ERROR] Alacritty encountered an unrecoverable error:

	[31mRusttype only supports TrueType fonts.
	Please select a TrueType font instead.[0m

```

**Comment by chrisduerr** [comment]
This has been fixed in #2224.
